assets/training/finetune_acft_image/environments/acft_image_huggingface/context/requirements.txt

```diff
@@ -1,7 +1,7 @@
-azureml-metrics[image]==0.0.37
+azureml-metrics[image]~=0.0.37
 azureml-evaluate-mlflow=={{latest-pypi-version}}
 azureml-acft-accelerator=={{latest-pypi-version}}
-azureml-acft-common-components[image]==0.0.37
+azureml-acft-common-components[image]~=0.0.37
```

Comment:
We have checked that the env builds with this, right?

Reply:
Yes, the build is getting succeeded. https://github.com/Azure/azureml-assets/actions/runs/7244093415/job/19731870750?pr=1942